dds/DCPS/unique_ptr.h

```diff
@@ -19,8 +19,8 @@ template <typename T>
 class rv : public T {
     rv();
     ~rv();
-    rv(rv const&);
-    void operator=(rv const&);
+    rv(const rv &);
```

Comment:
What about the move alternatives at the moment ACE_HAS_CPP11 defined?

Reply:
If the compiler has C++11, std::unique_ptr can be used.  Allowing OpenDDS to switch between the two would be a useful improvement for a future PR.

Reply:
Created issue #721 to not forget this